.github/workflows/main.yml

```diff
@@ -215,22 +239,30 @@ jobs:
     - run: cd src/doc && mdbook build --dest-dir ../../target/doc
     - name: Run linkchecker.sh
       run: |
+        BRANCH=${{ needs.channel.outputs.CHANNEL }}
         cd target
-        curl -sSLO https://raw.githubusercontent.com/rust-lang/rust/master/src/tools/linkchecker/linkcheck.sh
+        curl -sSLO "https://raw.githubusercontent.com/rust-lang/rust/$BRANCH/src/tools/linkchecker/linkcheck.sh"
         sh linkcheck.sh --all --path ../src/doc cargo
 
+  # Jobs here must be skipped if they aren't going to merge into master (nightly).
```

Comment:
Can this comment also explain more about the reason this exists? That is, saying something like "This is needed because GitHub Actions treats `success()` as false if a job is skipped, and the bors success/failure jobs below need to ignore skipped jobs."

Reply:
Now it looks better. Thanks for the suggestion!